cdap-api/src/main/java/co/cask/cdap/api/common/RuntimeArguments.java

```diff
@@ -29,6 +30,8 @@
 public final class RuntimeArguments {
 
   public static final Map<String, String> NO_ARGUMENTS = Collections.emptyMap();
+  private static final String STAR = "*";
```

Comment:
No liking as such :).. thought its more natural to read  `String wildCardPrefix = scope + DOT + STAR + DOT;` with `STAR`. Please let me know in case change is needed.